src/report/mod.rs

```diff
@@ -57,6 +57,13 @@ pub struct CrateResult {
     runs: [Option<BuildTestResult>; 2],
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+pub enum ReportPriority {
+    Low,
+    Medium,
+    High,
```

Comment:
I'm not sure "Low" / "Medium" / "High" is the right naming for this, as it'd get kinda messy if we need to add another priority. I'm not sure what naming scheme to switch though.